sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -228,20 +220,15 @@ Also, note that `get_wi_data` cannot return a fixed size array length because th
 
 2- SG size is not generally fixed.
 
-3- AMX has the flexibility of allowing variable sizes on the matrix (`dynamic_extent`).
-
-In the case of CUDA backend which is SYCL AOT compiled and SG size = 32 known and fixed, the additional marray capability will be provided.
+In the case of CUDA backend which is SYCL AOT compiled and SG size = 32 known and fixed, an additional `marray`-based capability is provided in [matrix Tensor Cores additional extension](sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix_cuda.asciidoc).
 
 The code listing below shows a synopsis of these new APIs.
 
 ```c++
 namespace sycl::ext::oneapi::experimental::matrix {
-template <typename T, size_t NumRows, size_t NumCols,
-          use Use, layout Layout,
-          typename Group = sycl::sub_group>
-struct joint_matrix {
-   wi_data<T, Use, NumRows, NumCols, Layout, Group> get_wi_data();
-};
+template <typename Group, typename T, size_t NumRows, size_t NumCols, use Use, layout Layout>
+   wi_data<group, T, Use, NumRows, NumCols, Layout> get_wi_data(Group sg, joint_matrix<Group, T, Use, Rows, Cols, Layout> Mat);
```

Comment:
```suggestion
   wi_data get_wi_data(Group sg, joint_matrix<...> Mat);
```

Reply:
For the joint_matrix ones the user has to provide them, so they should be documented. However I think that in some cases it is appropriate to omit them when they joint_matrix is used as an argument in other functions: using the <...> notation instead.

For wi_data I think it can just be made clear that it takes the joint_matrix as an argument in its constructor. Then it is quite clear to me that its construction will depend solely on the joint_matrix it takes (which is the case), and actually any mention of template arguments is not necessary IMO.